lib/conrad/emitter_queue.rb

```diff
@@ -0,0 +1,52 @@
+module Conrad
+  class EmitterQueue
+    include Singleton
+
+    attr_reader :background
+    attr_accessor :logger
```

Comment:
I don't see this logger being used anywhere. Was the plan to rescue somewhere that would consume it?